css/filetypes.scss

```diff
@@ -13,3 +13,11 @@
 .icon-filetype-draw {
 	background-image: url('../img/x-office-drawing.svg');
 }
+
+.icon-richdocuments {
+	background-image: url(../img/app-dark.svg);
+}
+
+body.dark .icon-richdocuments {
```

Comment:
- [ ] Double check if .dark is still a valid selector, likely has changed on server with the css refactoring

Reply:
You can use this in NC >= 25:
``` css
filter: var(--background-invert-if-dark);
```
and keep
``` css
body.dark .icon-richdocuments {
    background-image: url(../img/app.svg);
}
```
to stay compatible with NC < 25